src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: (($daySize + 2) * 7 + 10px);
+  gap: 1px;
+
+  &__day {
+    height: $daySize;
+    width: $daySize;
```

```suggestion
    height: $daySize;
    width: $daySize;
```
istead of this use @mixin